src/config.ts

```diff
@@ -27,5 +27,6 @@ export class JhiModuleConfig {
     i18nEnabled? = false;
     defaultI18nLang? = 'en';
     noi18nMessage? = 'translation-not-found';
+    i18nCacheBuster? = '';
```

Comment:
Should it be called something like `buildTimestamp` to eventually use it for something else than cache buster ?
(and hitting the cache is avoided only the first request after deployment)

Reply:
Fine with me. We could call it `buildTimestamp` for the config as well as the URL parameter.